packages/c/canvg.json

```diff
@@ -17,13 +17,13 @@
   "license": "MIT",
   "homepage": "https://canvg.github.io/canvg/examples/index.htm",
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/canvg/canvg.git",
+    "source": "npm",
+    "target": "canvg",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "lib",
         "files": [
-          "+(canvg|rgbcolor)*.+(js|map|css)"
+          "umd.+(js|js.map)"
```

Comment:
```suggestion
          "umd.js?(.map)"
```
`.+(js|js.map)` is somewhat redundant (should actually be `+(.js|.map)` to make use of `+` correctly), but we can just use `.js?(.map)` to specifically match `.js` or `.js.map`. [Glob](https://www.digitalocean.com/community/tools/glob?c=Y29tbWVudHM9dHJ1ZSZnbG9iPWxpYiUyRnVtZC5qcyUzRiUyOC5tYXAlMjkmbWF0Y2hlcz1mYWxzZSZ0ZXN0cz0lMkbFAzBJbXBvcnRlZCUyMGZyb20lMjBjYW52ZyUyME5QTSUyMHBhY2thZ8g3Q0hBTkdFTE9HLm1kxxNMSUNFTlNFxw5SRUFETUXKHsc%2FLmpzb27HE%2BYAqUJvdW5kaW5nQm94LmQudHPdHeQAys0hQ%2BQApNI4yhfRMkZvbnTSMckW0TBNb3VzZdIxyhfRMlBhcnNlctIzyxjTNHRo2jjNHNI8b2n0ANPKF9Iycm9wZXJ0edM1zBrROFNWR%2BQBKUxvYWT0AKnSH9JCY3JlZW7TO8oY0TRWaWV3UG9y8wDjzRrROGluZGV4LmJhYmVsLmrOOcYb0VDKF9dNZXPWSs8VdHlwZXPSX8oX0V%2FmA6%2FTE9EqRG9jdeQD8yUyRkFFbGXEC9Jq2CXdTm5pbWF0ZUNvbG9y31ndMN9kxDTfX9gr31rEL1RyYW5zZm9ybd9j3zTebENpcmNs%2FwDF2CrdWGxpcOQD7d9a2SzcXERlZnPfWNUo3lRzY99U1SjdVMcL3lHMJdxO%2FwCe0STeTGxpcHP%2FAffZK9xaRuYDRU1hdHJpeN9g3jHfZm9tcG9zaf8DgN4v3mJEcm9wU2hhZG9332PdMN5kR2F1c3NpYW5CbHX%2FBKjfMt9oTW9ycGhvbG9ned9m3TDdZGlsdGX%2FAMTYKt1Yb25031bVKN9UdEZhY%2F8CO9os3FxH31XSJd1ObHlw%2FwVD1yndVnJhZGll%2FwFY2yzcXEltYWf%2FAVnXKdxWTGlu31XWKN9UZWFy%2FwEM3zLfaMU2TWFya%2F8DGtkq3lhza99W1SjdVGlzc2luZ%2F8Cd98w32T%2FCBbZKN9UdGVybt9X2CvdWm9seWdv31rZK99aeWz%2FAtjcLNxcUmFkaWFs%2FwLc3zLfaMY2ZWPfXtYo3lRuZGVyZWTfWNks3FxTVv8FoNUn3VJ0b3DfU9Uo3lR5%2FwuT2CndVnltYm9s31fXKtxYVFJlZt9W1SjdVFNwYf8DdNcp3VZleP8CrtYo31R0Tm9k8xFW2CXfTnT%2FBSDdLN1caXT%2FAlbYKdxWVW5rbm93%2FwGs2SvdWv8MoNYn3FJlxijzEM7YJdxO9xF41SLRSOkPqOUHoeQM%2B9JK1yTdTFJvdGF08wKN1yTdTFNjYWzfS8oj3kprZXffScgi30hld1jfSckj30pld1nfSskj3UrJDN5OziffUmFuc2z%2FAcfQJ91S9wKr1iPdSvcUDtYj0UpwcmVzZXRz%2BgCS1CHbRm71BXnTINtEb2Zmc%2FcVydgl2073ASDUIdFGdXRpbCUyRmRlYnXzGAfRHthA9wFb0R7YQG1hdGjZP8kd2D5zdHJpbvoAv8sf2kJ5bPQBRNIfxEI%3D)